### DISCUSSION TITLE
VM not created

### DISCUSSION THREAD

**Discussion by vashkovich** [Marshall-Hallenbeck/red_team_attack_lab#9] [Q&A] [answered]
After command
    vagrant up
error appears:
"Error while connecting to Libvirt: Error making a connection to libvirt URI qemu:///system?no_verify=1&keyfile=/root/.ssh/id_rsa:                                                                       
Call to virConnectOpen failed: Failed to connect socket to '/var/run/libvirt/libvirt-sock': No such file or directory "
usage
 vagrant provision $host
doesn't help

How to fix the error?

**Comment by avi8892** [comment]
@vashkovich I have got the same issue.

**Comment by Marshall-Hallenbeck** [comment, marked as answer]
What OS are you running on, and do you have Virtualbox installed? 

What does `vagrant status` output?

Right now the only provider that works is Virtualbox, and it looks like this is trying to use Libvirt or something?

**Comment by vashkovich** [comment]
@Marshall-Hallenbeck  thanks for the tip.
After reinstall VirtualBox  and  Extension Pack
the problem is gone
